doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -164,7 +169,7 @@ is described in Chapter :ref:`syntaxextensionsandinterpretationscopes`.
                     : ( `name` [: `term`] := `term` )
                     : ' `pattern`
    name             : `ident` | _
-   qualid           : `ident` | `qualid` `access_ident`
+   qualid           : `ident` | `qualid` `field`
```

Comment:
I can get that "access identifier" is not a terrible name, but "field identifier" does not seem a very good name either because to me the notion of "field" is attached to the notion of "record" but here it is not a field of a record but a member of a module. Anyway, I wonder what is the whole point of defining this token in the first place and taking such length to explain it when all it is used for is in the definition of a `qualid`. I would remove entirely the notion of `access_ident` from the manual and instead change the definition of `qualid` into:

```suggestion
   qualid           : `ident` | `qualid`.`ident`
```

Reply:
I chose `field` because the token symbol in the code is `FIELD`.

It makes some sense to define this as a token because there can't be a space after the period.  `. foo` is 2 tokens.  What's a better name?  Qualifier??  qualid_component?

Reply:
There cannot be a space before either, can there?

Reply:
A space is permitted before.  The ident in a qualid is a different token.  Try it.

```
field: [
| FIELD
]

fields: [
| field fields
| field
]

fullyqualid: [
| ident fields
| ident
]
```

Reply:
Woah `Check Coq .Init .Datatypes .nat.` actually works. It just blows my mind. Is that actually wanted @herbelin?

Reply:
I don't know if it is the absolute best choice, but it is how it is implemented. One can argue for instance that it might be convenient when a qualified name is too long to hold in one line.

As a comparison, note that you can write `Pervasives . eq` in ocaml.